### PR TITLE
Feature/add behaviours description

### PR DESCRIPTION
**Pull request by uaihebert**
add description for CORS filters and Spring PostConstruct.

### CODE REVIEW

File: docs/src/main/asciidoc/http-reference.adoc [hunk 1]
```diff
@@ -153,6 +153,15 @@ quarkus.http.cors.exposed-headers=Content-Disposition
 quarkus.http.cors.access-control-max-age=24H
 ----
 
+[TIP]
+====
+When CORS functionality is activated your filters - e.g. ContainerRequestFilter or ContainerResponseFilter - will not be invoked.
```
Comment (gsmet) [inline docs/src/main/asciidoc/http-reference.adoc]: I'm not sure it's precise enough.

I suppose they are not enabled for the CORS requests but they are still enabled for standard requests? If not, I don't think this is expected.

**Review by gsmet — CHANGES_REQUESTED** [review]
Asked a question we need to sort out before thinking about merging.

I also think it should be rewritten to be:
- more precise for the concerned requests
- more general for the consequences: a filter can do a lot more things than adding headers.

**Comment by gsmet** [comment]
BTW, sorry, I'm in serial review mode but thanks for taking the time to contribute :).

File: docs/src/main/asciidoc/http-reference.adoc (hunk 1 above)
Comment (uaihebert) [reply docs/src/main/asciidoc/http-reference.adoc]: What I experienced with quarkus is. 
When there are no CORS your filters will be invoked for every request.

When you got CORS your filters are not used for CORS request. That is what I am trying to explain here.

**Comment by uaihebert** [comment]
no problem at all. I just want to make it clear some behaviors that were not described.

**Comment by uaihebert** [comment]
> Asked a question we need to sort out before thinking about merging.
> 
> I also think it should be rewritten to be:
> 
> * more precise for the concerned requests
> * more general for the consequences: a filter can do a lot more things than adding headers.

Indeed can do more than add headers, I just added the headers as an example. A lot of companies requires that some header should always be present.

File: docs/src/main/asciidoc/http-reference.adoc (hunk 1 above)
Comment (uaihebert) [reply docs/src/main/asciidoc/http-reference.adoc]: ps. I am just trying first to detail what I am trying to cover to hear from you. I will improve the text later! :D

**Comment by geoand** [comment]
@gsmet when your concerns have been addressed please let me know so I can give this a go as well.
Thanks

**Comment by gsmet** [comment]
Closing for now. Please open another PR taking the comments into account if you wish to.

Thanks.

**Pull request closed without merging**